sklearn/naive_bayes.py

```diff
@@ -259,6 +278,9 @@ def _update_mean_variance(n_past, mu, var, X, sample_weight=None):
 
         return total_mu, total_var
 
+    @deprecated("The classes argument will be removed in version 0.21."
+                "A new classes argument has been added as a class parameter"
+                "since version 0.19 which is recommended to be used now.")
```

Comment:
@amueller - When I add this deprecated message, I start getting an error:

> ERROR: /home/travis/build/scikit-learn/scikit-learn/sklearn/tests/test_common.py.test_non_meta_estimators:check_fit_score_takes_y(GaussianNB)
> ----------------------------------------------------------------------
> Traceback (most recent call last):
>   File "/home/travis/miniconda/envs/testenv/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
>     self.test(*self.arg)
>   File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/testing.py", line 775, in __call__
>     return self.check(*args, **kwargs)
>   File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/testing.py", line 291, in wrapper
>     return fn(*args, **kwargs)
>   File "/home/travis/build/scikit-learn/scikit-learn/sklearn/utils/estimator_checks.py", line 820, in check_fit_score_takes_y
>     assert_true(args[1] in ["y", "Y"],
> IndexError: list index out of range

If I comment this deprecation, this error goes away, atleast in my local. Thoughts?